docs/data/sdks/flutter/index.md

```diff
@@ -419,7 +419,7 @@ These features aren't supported in Flutter web:
 
 - `enableCoppaControl`
 - `disableCoppaControl`
-- `trackingSessionEvents`
+- `trackingSessionEvents`. We are still tracking the sessionId in Flutter Web, but whether to automatically log start and end session events corresponding to the start and end of a user's session is not supported in Flutter Web.
```

Comment:
Do we even fire the session events in Flutter web with JS SDK? Maybe we could claim we don't support session events in Flutter web? The current wording feels like this control still work for Flutter web.